modules/srvrlss-using-cluster-logging-find-knative-serving-logs.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
+
+[id=using-cluster-logging-to-find-Knative-Serving-logs_{context}"]
+= Using Cluster Logging to find logs for Knative Serving components
+
+.Prerequisites
+
+* xref:../efk-logging-about_{context}[Cluster Logging components] are installed.
+
+.Procedure
+
+. To open the Kibana UI (the visualization tool for Elasticsearch), use the following command to get the Kibana route:
+
+`oc -n openshift-logging get route kibana`
+
+. Use the route's URL to navigate to the Kibana dashboard and log in.
+
+. Ensure the index is set to *.all*. If the index is not set to *.all*, only the OpenShift system logs will be listed. 
+
+. You can filter the logs by using the knative-serving namespace. Enter `kubernetes.namespace_name:knative-serving` in the search box to filter results.
++
+NOTE: Knative Serving uses structured logging by default. You can enable the parsing of these logs by customizing the xref:../efk-logging-fluentd-json_{context}[ClusterLogging fluentd settings]. This makes the logs more searchable and enables filtering on the log level to quickly identify issues.
```

Comment:
[NOTE]
`=` x 4 (same issue as with the code block `====`)
Knative Serving uses structured logging by default. You can enable the parsing of these logs by customizing the ClusterLogging fluentd settings. This makes the logs more searchable and enables filtering on the log level to quickly identify issues.
`=` x 4

I would suggest adding the link to appropriate assembly the additional resources section of the assembly this module will build in since we can't link module to module.